README.md

```diff
@@ -1,58 +1,71 @@
-# unified-codes
+# Universal Codes
 
-## Getting started
+Welcome! This is Universal Codes!
 
-1. Install dependencies: `npm install`.
-2. Follow application instructions below.
+This is a responsive web application over a GraphQL API.
 
-## Applications
+It is built using Typescript, with a React Frontend and an Apollo Server with a DGraph database. The frontend code is heavily 'borrowed' from the [Notify frontend](https://github.com/openmsupply/notify/tree/main/frontend)
 
-### data-loader
+The test framework is jest; functional areas are separated into packages and managed with [lerna](https://lerna.js.org/)
 
-Application for parsing product data and populating Dgraph database.
+## Usage
 
-1. Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md)).
-3. Populate graph schema and demo data: `npx nx serve data-loader`.
-4. Dgraph is now running, you can access at [http://localhost:9080](http://localhost:9080).
+- Install dependencies (Using node v20+ and yarn):
+  - `yarn install`
+- Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md))
+  - You can then use the [explorer](https://play.dgraph.io/?latest) (server URL is http://localhost:8080)
+- Load data into DGraph:
+  - `yarn load-data`
+- Upload the GraphQL schema to DGraph:
+  - `yarn update-dgraph-schema`
+- Start the data-service:
+  - `yarn start-data-service`
+  - Access v1 api at http://localhost:4000/v1/graphql
+  - Access v2 api at http://localhost:4000/v2/graphql
 
-### data-service
+### Frontend
 
-Backend GraphQL server for accessing product data.
+We are currently re-implementing the frontend. For the existing frontend:
 
-1. Run GraphQL server: `npx nx serve data-service`.
-2. Access v1 api at http://localhost:4000/v1/graphql  
-3. Access v2 api at http://localhost:4000/v2/graphql
+1. Run web server: `npx nx serve web`
```

Comment:
I get this. Not sure it's a big deal though. Are you expecting to to work with this branch though?
```npx nx serve web

 >  NX   The current directory isn't part of an Nx workspace.

   To create a workspace run:
   npx create-nx-workspace@latest <workspace name>
   
   To add Nx to an existing workspace with a workspace-specific nx.json, run:
   npx nx@latest init


 >  NX   For more information please visit https://nx.dev/```

Reply:
[9bc2942](https://github.com/msupply-foundation/unified-codes/pull/430/commits/9bc29428a278778c35cdbc8af8ee2cb494201a83)